cmds/ocm/topics/toi/bootstrapping/topic.go

```diff
@@ -332,7 +332,7 @@ execution and reading provided executor outputs after the execution.
 /
 └── toi
     ├── inputs
-    │   ├── config      config info from package specification
+    │   ├── config      info from package specification
```

Comment:
I would omit `info` instead of `config`: the file `config` contains the configuration from the package specification.

May be, replace it by `configuration from package specification`to avoid the repetition of `config`.

Reply:
```suggestion
    │   ├── config      configuration from package specification
```

Reply:
done.

Actions taken:
- applied suggested changes